Source/WebCore/history/CachedPage.h

```diff
@@ -66,6 +67,8 @@ class CachedPage final : public CanMakeCheckedPtr<CachedPage> {
     void markForContentsSizeChanged() { m_needsUpdateContentsSize = true; }
 
 private:
+    void restoreNavigationAPIHistoryItems(LocalFrame&, CheckedRef<BackForwardController>);
```

Comment:
The parameter should not be a `CheckedRef` here, this doesn't match our Safer CPP guidelines. Please pass in a raw pointer.